auth-service/src/test/kotlin/uk/gov/communities/delta/service/LDAPRepositoryTest.kt

```diff
@@ -0,0 +1,84 @@
+import io.mockk.*
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import uk.gov.communities.delta.auth.config.LDAPConfig
+import uk.gov.communities.delta.auth.repositories.LdapRepository
+import javax.naming.NamingEnumeration
+import javax.naming.directory.*
+import javax.naming.ldap.InitialLdapContext
+
+class LdapRepositoryTest {
+
+    private val ldapConfig = mockk<LDAPConfig>(relaxed = true)
+    private val ctx = mockk<InitialLdapContext>()
+    private val ldapRepository = LdapRepository(ldapConfig, LdapRepository.ObjectGUIDMode.NEW_JAVA_UUID_STRING)
+
+//    @Test
```

Comment:
Why's this commented out?

Reply:
I couldn't quite the test working but didn't want to spend too long behind it. Il remove it for now

Reply:
Could we add a ticket to re-instate and fix this removed test, if it's useful?

Reply:
Could we add a ticket to re-instate and fix this removed test, if it's useful?